hoomd/mpcd/collide.py

```diff
@@ -253,11 +264,7 @@ class AndersenThermostat(CollisionMethod):
     )
 
     def __init__(self, period, kT, embedded_particles=None):
-        super().__init__(period, embedded_particles)
-
-        param_dict = ParameterDict(kT=hoomd.variant.Variant)
-        param_dict["kT"] = kT
-        self._param_dict.update(param_dict)
+        super().__init__(period, kT, embedded_particles)
```

Comment:
Using keyword arguments will help insulate this against changing argument orders, if we reorder in future:
```suggestion
        super().__init__(period=period, kT=kT, embedded_particles=embedded_particles)
```